code/modules/cm_marines/equipment/kit_boxes.dm

```diff
@@ -212,6 +216,17 @@
 				return FALSE
 			return TRUE
 
+/obj/item/spec_kit/rifleman/can_use(mob/living/carbon/human/user)
+	if(!length(allowed_roles_list))
+		return TRUE
+
+	for(var/allowed_role in allowed_roles_list)
+		if(user.job == allowed_role)//Alternate check to normal kit as this is distributed to people without SKILL_SPEC_TRAINED.
+			if(skillcheck(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_KITTED) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
```

Comment:
What about SGs, on purpose?

Reply:
Hadn't thought of SGs, but I think they're fine to be left out same as kitted specs.